content/blog/one-year-of-excalidraw/index.md

```diff
@@ -103,6 +103,9 @@ One of the last things we've introduced this year was a completely new npm packa
 - React Europe talk: https://www.youtube.com/watch?v=fix2-SynPGE
 - Used to illustrate an O'Reilly Book: https://www.amazon.com/dp/1492057096
 - Another book: https://twitter.com/dchest/status/1264237749642637312
+- Few companies also integrated Excalidraw in the product
+  - [HackerRank](https://blog.hackerrank.com/virtual-whiteboarding-for-system-design-interviews/)
```

Comment:
Should we replace this with [twitter](https://twitter.com/hackerrank/status/1260674103138082816) link